invenio_rdm_records/fixtures/demo.py

```diff
@@ -312,3 +312,61 @@ def create_fake_record():
     }
 
     return json.loads(json.dumps(data_to_use))
+
+
+def create_fake_community():
+    """Create minimal community for demo purposes."""
+    fake = Faker()
+    return {
+        "id": fake.unique.domain_word(),
+        "access": {
+            "visibility": "public",
+        },
+        "metadata": {
+            "title": fake.sentence(nb_words=5, variable_nb_words=True),
+            "type": random.choice([
+                "organization", "event", "topic", "project"
+            ]),
+        }
+    }
+
+
+def create_fake_draft():
```

Comment:
don't we already have a `fake_data` record used in the `demo` func?

Reply:
@Pineirin this comment was not addressed